nifi-api/src/test/resources/TestVariableRegistry/foobar.properties

```diff
@@ -0,0 +1 @@
+fake.property.3=test me out 3, test me out 4
```

Comment:
I believe this file should have the ASF license information in a header. This would also alleviate the need to add the rat-check exclusions in the pom.xml